bug(ios): 'getConfigValue' is deprecated

**Plugin version:**
- 2.0.0


**Platform(s):**
- iOS


**Current behavior:**

When running app on simulator or physical device, the following warning message will occur on Xcode:

capacitor-plugin-demo/node_modules/@capawesome/capacitor-badge/ios/Plugin/BadgePlugin.swift:117:26: warning build: 'getConfigValue' is deprecated: use getConfig() and access config values using the methods available depending on the type.

<img width="1776" alt="Screen Shot 2022-10-04 at 18 46 19" src="https://user-images.githubusercontent.com/16042676/193803504-1e58a555-2de0-47e5-b568-e691f637bf2f.png">



**Expected behavior:**

The app should run without warning.

**Steps to reproduce:**

1. clone https://github.com/robingenz/capacitor-plugin-demo
2. `npm i`
3. `npx ionic cap sync ios`
4. `npx ionic cap open ios`
5. Use Xcode to start running app on simulator or physical device


**Related code:**
<!-- Code which helps to reproduce or better understand the issue. -->

https://github.com/capawesome-team/capacitor-badge/blob/2e796daa56da6dc8caa3d0ec564c2c6b9b75b0a0/ios/Plugin/BadgePlugin.swift#L114-L125


**Capacitor doctor:**

```
💊   Capacitor Doctor  💊

Latest Dependencies:

  @capacitor/cli: 4.3.0
  @capacitor/core: 4.3.0
  @capacitor/android: 4.3.0
  @capacitor/ios: 4.3.0

Installed Dependencies:

  @capacitor/cli: 4.3.0
  @capacitor/core: 4.3.0
  @capacitor/android: 4.3.0
  @capacitor/ios: 4.3.0

[success] iOS looking great! 👌
[success] Android looking great! 👌
```

--- a/ios/Plugin/BadgePlugin.swift
+++ b/ios/Plugin/BadgePlugin.swift
@@ -114,12 +114,8 @@ public class BadgePlugin: CAPPlugin {
     private func badgeConfig() -> BadgeConfig {
         var config = BadgeConfig()
 
-        if let persist = getConfigValue("persist") as? Bool {
-            config.persist = persist
-        }
-        if let autoClear = getConfigValue("autoClear") as? Bool {
-            config.autoClear = autoClear
-        }
+        config.persist = getConfig().getBoolean("persist", config.persist)
+        config.autoClear = getConfig().getBoolean("autoClear", config.autoClear)
 
         return config
     }

